source/walkthroughs/basics/meteor/preparing_sample_app.md.erb

```diff
@@ -24,19 +24,19 @@ In the previous step, you only created a Meteor app in development mode. In this
     In order to provide you appropriate instructions, please choose your Meteor version: <%= render_partial("/shared/meteor_picker.html") %>
 </p>
 
-Inside the `leaderboard` directory, use the <code class="meteor old">meteor bundle</code><code class="meteor new">meteor build</code> command to create a package tarball.
+Inside the `leaderboard` directory, use the <code class="meteor v13">meteor bundle</code><code class="meteor v14 v16">meteor build</code> command to create a package tarball.
```

Comment:
The choice between 1.4 and 1.6 makes no difference in this page, which means that users of 1.4 will have to perform a useless click. The picker here should be <= 1.3.5.1 and >= 1.4.

Reply:
@OnixGH the picker is a shared picker which I simply extended to cover the 3-version case. So to make it clear, do you want me to create a different shared picker for these ones that 2 options suffice for?

Reply:
That's one solution, yep. I don't mind a little extra code for user-friendliness :)